### ISSUE TITLE
Feature Request: Option to have images scale to window

### ISSUE THREAD

**Issue by betsythefc** [MacDownApp/macdown#1311]
Hello! Currently, if a large image is in a document, it extends beyond the size of the window when rendered, making then hard to see. An option to keep images wider than current window to be rendered within the margins would improve rendered readability.

**Comment by granludo** [comment]
Hi, can we get the osx spellcheck on the editor? 
Thanks

**Comment by kleingeld1** [comment]
Yes please, this is the one feature why I am considering switching my main editor. Please add this.
